modules/core/src/main/java/org/apache/synapse/config/xml/rest/APIFactory.java

```diff
@@ -157,6 +165,15 @@ public static API createAPI(OMElement apiElt, Properties properties) {
         return api;
     }
 
+    private static void validateBindsTo(API api, Resource resource) {
```

Comment:
let's rename this method. This is not only validating, but also adding bindings

Reply:
This method is not used anymore, it has been moved to `ApiUtils`, where only validation is done. Removed this in 89315e2 and cc02c4c